### DISCUSSION TITLE
import langroid.language_models → ERROR

### DISCUSSION THREAD

**Discussion by philchass** [langroid/langroid#389] [General]
Using Python 3.11.5
pip install langroid → OK

Using IPython :
import langroid as lr → OK
import langroid.language_models as lm →  ModuleNotFoundError: No module named 'langroid.language_models'; 'langroid' is not a package

I'm stuck. Please help

**Comment by pchalasani** [comment]
@philchass I don't see this error when I try it. If you give more details on how you installed langroid, virtual env, etc, I can look into it.

**Comment by philchass** [comment]
I'm sorry Prasad, my Python env was not installed correctly, it works now.
Do you have an idea to make langroid working with a ollama local LLM + pandas + a CSV file to interact with?

**Reply by pchalasani** [reply]
see this table chat example: https://github.com/langroid/langroid-examples/blob/main/examples/data-qa/table_chat.py
